plugin.yml

```diff
@@ -23,6 +23,30 @@ configuration:
     build-alias:
       type: [ string, array ]
       minimum: 1
+    builder:
+      type: object
+      properties:
+        bootstrap:
+          type: boolean
+        create:
+          type: boolean
+        debug:
+          type: boolean
+        driver:
+          type: string
+          enum: [ "docker-container", "kubernetes", "remote" ]
+        driver-opt:
+          type: string
+        name:
+          type: string
+        platform:
+          type: string
+        remote-address:
+          type: string
+        remove:
+          type: boolean
+        use:
+          type: string
```

Comment:
I would suggest adding `additionalProperties: false` to restrict valid elements to exactly those listed